appveyor.yml

```diff
@@ -38,17 +38,23 @@ before_test:
 
 test_script:
   # The sleep is needed to allow the Appveyor tracing to start correctly after a reboot (locale change)
-  - ps: Start-Sleep -Seconds 5
-  - SET
-  - chcp
-  - ps: Get-WinSystemLocale
-  - SET PATH=C:\%PLATFORM%\bin;%PATH%
-  - SET LOG_SPEC_ORDER=true
-  - bundle install --jobs 4 --retry 2 --without development extra
-  - type Gemfile.lock
-  - ruby -v
-  - bundle exec rake parallel:spec[2]
-
+  - ps: |
+      Start-Sleep -Seconds 5
+      chcp
+      Get-WinSystemLocale
+      $Env:PATH = "C:\${Env:PLATFORM}\bin;${Env:PATH}"
+      $Env:LOG_SPEC_ORDER = 'true'
+      Get-ChildItem Env: | % { Write-Output "$($_.Key): $($_.Value)"  }
+      bundle install --jobs 4 --retry 2 --without development extra
+      Get-Content Gemfile.lock
+      ruby -v
+      gem --version
+      bundle --version
+      bundle exec rake parallel:spec[2]
+      $exitCode = $LASTEXITCODE
+      Write-Output "Spec run exited with $exitCode"
+      # exit $exitCode
+      $host.SetShouldExit($exitCode)
```

Comment:
So to get an exit status to propagate correctly from PowerShell code run in AppVeyor, we have to call against the `$host` like the above.  `exit #` does not work properly within their custom host as it does with the Puppet PowerShell module.

/cc @glennsarti @MikaelSmith @jpogran